docs/client-concepts/certificates/working-with-certificates.asciidoc

```diff
@@ -115,7 +115,7 @@ the local CA certificate is part of the chain that was used to generate the serv
 ==== Client Certificates
 
 X-Pack also allows you to configure a {xpack_current}/pki-realm.html[PKI realm] to enable user authentication
-through client certificates. The {ref_current}/certutil.html[+certutil+ tool] included with X-Pack allows you to

```

Comment:
Is this the correct way ? We broke the doc build on this before not sure if this ends up reverting it?

Reply:
That link looks fine there, but I'll double check. If I get a chance today, I'll take a look at AsciiDocNet